### PR TITLE
fix(instrumentation-runtime-node): use finer GC duration buckets

### PR DESCRIPTION
**Pull request by mcollina**
## Which problem is this PR solving?

- `v8js.gc.duration` used very coarse bucket boundaries (`[0.01, 0.1, 1, 10]`), which caused sub-second GC pauses like ~200ms to be reported in the `<=1s` bucket.

## Short description of the changes

- switch the GC duration histogram to latency-style second buckets
- add a unit test covering the configured bucket boundaries

## Validation

- `npm run version:update -w @opentelemetry/instrumentation-runtime-node`
- `npm run compile -w @opentelemetry/instrumentation-runtime-node`
- `npm test -w @opentelemetry/instrumentation-runtime-node`

### CODE REVIEW

**Comment by pichlermarc** [comment]
@maryliag @trentm I seem to remember one (or both) of you being involved on [the Semantic Conventions side](https://github.com/open-telemetry/semantic-conventions/blob/cbb10b68a58c91a330ce8c9c66dd1a5195c6a886/docs/runtime/v8js-metrics.md#metric-v8jsgcduration) for this metric definition. WDYT?

I think having more granular buckets could be helpful for this metric. Introducing more buckets is usually not a breaking change.

**Comment by maryliag** [comment]
that was indeed me! I see no issue with this change, but I would ask you @mcollina to update the semantic convention first [here](https://github.com/open-telemetry/semantic-conventions/blob/cbb10b68a58c91a330ce8c9c66dd1a5195c6a886/docs/runtime/v8js-metrics.md?plain=1#L29), then we can go ahead with this PR!
Feel free to tag me there once the PR is up

File: packages/instrumentation-runtime-node/src/metrics/gcCollector.ts [hunk 1]
```diff
@@ -11,7 +11,11 @@ import { Histogram, ValueType } from '@opentelemetry/api';
 import { BaseCollector } from './baseCollector';
 import { ATTR_V8JS_GC_TYPE, METRIC_V8JS_GC_DURATION } from '../semconv';
 
-const DEFAULT_GC_DURATION_BUCKETS = [0.01, 0.1, 1, 10];
+// Use latency-style second buckets so sub-second GC pauses do not all collapse
+// into the `<=1s` bucket in histogram-based backends.
```
Comment (maryliag) [inline packages/instrumentation-runtime-node/src/metrics/gcCollector.ts]:
```suggestion
```

No need to add this comment, since after you make the semantic conventions change, it would be just following it

**Comment by mcollina** [comment]
> I think having more granular buckets could be helpful for this metric. Introducing more buckets is usually not a breaking change.

It's not just helpful, the current buckets leads to bad decisions. 

A major gc break stop the event loop. There is a massive of difference between a 100ms major gc break and a 1s major gc break.
One is ok, the other one cause a measurable production incident and should raise alerts.

**Review by raphael-theriault-swi — APPROVED** [review]
Pending on semconv update

**Comment by joaopgrassi** [comment]
Took the liberty to fix missing stuff on the semconv PR. It is merged now.

**Review by pichlermarc — APPROVED** [review]
